examples/wallet/package.json

```diff
@@ -164,135 +164,135 @@
-    "eslint-plugin-jsx-a11y": "6.2.1",
-    "eslint-plugin-prettier": "3.0.1",
-    "eslint-plugin-promise": "4.1.1",
-    "eslint-plugin-react": "7.12.4",
+    "enzyme": "3.11.0",
+    "enzyme-adapter-react-16": "1.15.2",
+    "enzyme-to-json": "3.4.3",
+    "eslint": "6.8.0",
+    "eslint-config-airbnb": "18.0.1",
+    "eslint-config-erb": "0.1.1",
+    "eslint-config-prettier": "6.9.0",
+    "eslint-formatter-pretty": "3.0.1",
+    "eslint-import-resolver-webpack": "0.12.0",
+    "eslint-plugin-compat": "3.3.0",
+    "eslint-plugin-flowtype": "4.5.3",
+    "eslint-plugin-import": "2.19.1",
+    "eslint-plugin-jest": "23.2.0",
+    "eslint-plugin-jsx-a11y": "6.2.3",
+    "eslint-plugin-prettier": "3.1.2",
+    "eslint-plugin-promise": "4.2.1",
+    "eslint-plugin-react": "7.17.0",
     "eslint-plugin-testcafe": "0.2.1",
     "fbjs-scripts": "1.2.0",
-    "file-loader": "3.0.1",
-    "flow-bin": "0.95.1",
+    "file-loader": "5.0.2",
+    "flow-bin": "0.114.0",
     "flow-coverage-report": "0.6.1",
     "flow-runtime": "0.17.0",
-    "flow-typed": "2.5.1",
-    "husky": "1.3.1",
+    "flow-typed": "2.6.2",
+    "husky": "3.1.0",
     "identity-obj-proxy": "3.0.0",
-    "jest": "24.5.0",
-    "lint-staged": "8.1.5",
-    "mini-css-extract-plugin": "0.5.0",
-    "node-sass": "4.11.0",
+    "jest": "24.9.0",
+    "lint-staged": "9.5.0",
+    "mini-css-extract-plugin": "0.9.0",
+    "node-sass": "4.13.0",
     "opencollective-postinstall": "2.0.2",
-    "optimize-css-assets-webpack-plugin": "5.0.1",
-    "prettier": "1.16.4",
-    "react-test-renderer": "16.8.6",
+    "optimize-css-assets-webpack-plugin": "5.0.3",
+    "prettier": "1.19.1",
+    "react-test-renderer": "16.12.0",
     "redux-logger": "3.0.6",
-    "rimraf": "2.6.3",
-    "sass-loader": "7.1.0",
-    "sinon": "7.3.1",
-    "spectron": "5.0.0",
-    "style-loader": "0.23.1",
-    "stylelint": "9.10.1",
-    "stylelint-config-prettier": "5.0.0",
-    "stylelint-config-standard": "18.2.0",
-    "terser-webpack-plugin": "1.2.3",
-    "testcafe": "1.1.0",
-    "testcafe-browser-provider-electron": "0.0.8",
+    "rimraf": "3.0.0",
+    "sass-loader": "8.0.0",
+    "sinon": "8.0.2",
+    "spectron": "9.0.0",
+    "style-loader": "1.1.2",
+    "stylelint": "12.0.1",
+    "stylelint-config-prettier": "8.0.0",
+    "stylelint-config-standard": "19.0.0",
+    "terser-webpack-plugin": "2.3.1",
+    "testcafe": "1.7.1",
+    "testcafe-browser-provider-electron": "0.0.13",
     "testcafe-live": "0.1.4",
-    "testcafe-react-selectors": "3.1.0",
-    "url-loader": "1.1.2",
-    "webpack": "4.29.6",
+    "testcafe-react-selectors": "3.3.0",
+    "url-loader": "3.0.0",
+    "webpack": "4.41.5",
     "webpack-bundle-analyzer": "3.6.0",
-    "webpack-cli": "3.3.0",
-    "webpack-dev-server": "3.2.1",
-    "webpack-merge": "4.2.1",
-    "yarn": "1.19.1"
+    "webpack-cli": "3.3.10",
+    "webpack-dev-server": "3.10.1",
+    "webpack-merge": "4.2.2",
+    "yarn": "1.21.1"
```

Comment:
I had to use node 12 to install the dependencies

Reply:
whaaaaat? doesn't it work with node 10?